### PR TITLE
Zhone health

### PR DESCRIPTION
**Pull request by jerji**
This pull request adds CPU and memory to the Zhone devices. it also makes the number of ports more manageable as LibreNMS would crash with all the logical ports active.

The original support for this OS was asked by [@jlixfeld](https://github.com/jlixfeld) which is currently employed at Beanfield.

I could not add the processor names because of the way processor is implemented. It does not seem to support pre-cache

I hope you enjoy my code ^-^

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by PipoCanaja** [comment]
Hi @jerji 
Thanx for your contribution and welcome here. 
Could you please check the tests and add the necessary files to this PR ? If you need help you can ask here or in the Discord.

**Comment by jerji** [comment]
@PipoCanaja I've added test data but something broke.

**Comment by PipoCanaja** [comment]
Hi @jerji 
You are almost done. You need to follow the procedure and generate + attach the JSON file as well to this PR. That will fix the regression tests.

**Comment by jerji** [comment]
Hey sorry to tag you again @PipoCanaja but I don't understand this error :(

File: includes/discovery/ports/slms.inc.php
Comment (murrant) [inline includes/discovery/ports/slms.inc.php:1]: I'd prefer to avoid yet another custom port polling implementation if we could.  Can you just set bad_if on the OS instead?

**Comment by murrant** [comment]
Also, you've attached an empty .json test data file. See the link from the bot for more info.

**Comment by jerji** [comment]
The only test that failed is not from this OS

**Comment by jerji** [comment]
@murrant @PipoCanaja 
I'm getting the same output as when I didn't have Test data but I have some now.

**Comment by jerji** [comment]
Any update on this?

**Comment by murrant** [comment]
Bizarre failure, I'm not sure what to make of it.

**Comment by jerji** [comment]
@murrant  Is there any way I could troubleshoot this?

**Comment by jerji** [comment]
Woohoo :D got the tests to pass

**Comment by PipoCanaja** [comment]
Hi @jerji 
Looks good to me. Will wait for a review from @murrant, for the PHP code in the port module.

**Review by Jellyfrog — APPROVED** [review]

**Pull request merged**
